proposals/bulk-memory-operations/Overview.md

```diff
@@ -282,8 +287,8 @@ implemented as follows:
 (func $start
   (if (get_global 0)
 
-    ;; copy data segment 1 into memory
-    (memory.init 1
+    ;; copy data segment 1 into memory 0
+    (memory.init 1 0
```

Comment:
I think we'll probably omit the memory indexes (same as `memory.grow`) until we have multiple memories.